resolwe_bio/processes/alignment/bowtie.yml

```diff
@@ -140,14 +140,14 @@
       reporting={{reporting.r}}
 
       {% for r in reads.fastq %}
-        READS_NAME=`basename {{ r.file }} .fastq.gz`
+        READS_NAME={{ r.file | basename | replace(".fastq.gz", "") | replace(" ", "") }}
```

Comment:
If we want to handle filenames containing spaces by removing spaces, it would be better to do it systematically via input sanitizer for all processes. I suggested this some time ago but the general opinion at the time was that filenames should be conserved, including spaces. Consider [this approach](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/reads_processing/bbduk.yml#L410).